core/debug/debug.c

```diff
@@ -4,7 +4,7 @@
 
 volatile bool in_debugger = false;
 
-CEMU_INLINE uint8_t debug_port_read_byte(const uint32_t addr) {
```

Comment:
Oh well, we can probably rely on the compiler to inline whatever it deems necessary.
So let's just remove all `inline`, as @Vogtinator proposed.